python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,889 @@
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "blurMultiplier", defaultValue = 1.0, minValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "computedFocalLength", direction = Gaffer.Plug.Direction.Out, defaultValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "horizontalAperture", defaultValue = 1.0, minValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "computedHorizontalAperture", direction = Gaffer.Plug.Direction.Out, defaultValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "computedFStop", direction = Gaffer.Plug.Direction.Out, defaultValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.IntPlug( "maxBlurRadius", defaultValue = 256, minValue = 2, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "computedFocalLengthWorldScale", direction = Gaffer.Plug.Direction.Out, defaultValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "depthPlaneScalingFactor", defaultValue = 0.5, minValue = 0.10000000149011612, maxValue = 0.8999999761581421, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( GafferImage.Merge( "Merge26" ) )
+__children["FocalBlur"]["Merge26"]["in"].resize( 3 )
+__children["FocalBlur"]["Merge26"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "approximationThreshold", defaultValue = 0.0010000000474974513, minValue = 0.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.Dot( "Dot15" ) )
+__children["FocalBlur"]["Dot15"].setup( GafferImage.ImagePlug( "in", ) )
+__children["FocalBlur"]["Dot15"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.Dot( "Dot16" ) )
+__children["FocalBlur"]["Dot16"].setup( GafferImage.ImagePlug( "in", ) )
+__children["FocalBlur"]["Dot16"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["FocalBlur"].addChild( Gaffer.FloatPlug( "backgroundZValue", defaultValue = 100000.0, flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
```

Comment:
Seeing as we've renamed the other "Z" plugs to "Depth", should this be "backgroundDepthValue" or "backgroundValue"?